src/azure-cli/azure/cli/command_modules/acs/tests/latest/recordings/test_aks_abort.yaml

```diff
@@ -13,10 +13,9 @@ interactions:
       ParameterSetName:
       - --resource-group --name --ssh-key-value --no-wait
       User-Agent:
-      - AZURECLI/2.49.0 azsdk-python-azure-mgmt-containerservice/24.0.0 Python/3.8.10
-        (Linux-5.15.0-1039-azure-x86_64-with-glibc2.29)
+      - AZURECLI/2.57.0 azsdk-python-core/1.28.0 Python/3.11.5 (macOS-14.3-x86_64-i386-64bit)
```

Comment:
probably you don't need to update this file

Reply:
removed these changes